tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py

```diff
@@ -0,0 +1,79 @@
+"""
+This guide demonstrates how to use Great Expectations in a quick-start
+notebook environment with SQL. It is useful to explore and understand how Great Expectations
+works, using your own data.
+
+Note:
+- Do not follow this workflow when you are building a durable workflow because it uses
+ephemeral assets.
+- Do not use this workflow for embedding in a pipeline or CI system, because it uses an
+iterative process for trying and refining expectations.
+"""
+
+import pathlib
+import shutil
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py import_gx">
+import great_expectations as gx
+import great_expectations.expectations as gxe
+
+# </snippet>
+
+# Set up
+# NOTE: Context is a singleton now. Once the context has been set instantiated in a session
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py get_context">
+context = gx.get_context()
+# </snippet>
+
+sqlite_database_path = pathlib.Path(
+    gx.__file__,
+    "..",
+    "..",
+    "tests",
+    "test_sets",
+    "quickstart",
+    "yellow_tripdata.db",
+).resolve(strict=True)
+
+assert sqlite_database_path.exists()
+
+# Copy the test database to the current directory for purposes of this test file (users will curl)
+shutil.copy(sqlite_database_path, "yellow_tripdata.db")
+
+
+# <snippet name="tests/integration/docusaurus/tutorials/quickstart/v1_sql_quickstart.py connect_to_data">
+# curl https://raw.githubusercontent.com/great-expectations/great_expectations/develop/tests/test_sets/quickstart/yellow_tripdata.db > yellow_tripdata.db
+connection_string = "sqlite:///yellow_tripdata.db"
+batch = context.sources.pandas_default.read_sql(
+    "SELECT * FROM yellow_tripdata_sample_2022_01", connection_string
```

Comment:
Needed to modify `yellow_tripdata.db` because when I uploaded the CSV into the DB, I didn't consider the fact that types weren't implicitly coerced.

I used this flow to fix: https://stackoverflow.com/a/10385666